libmuscle/python/libmuscle/instance.py

```diff
@@ -965,12 +1015,20 @@ def __receive_settings(self) -> bool:
         self._trigger_manager.harmonise_wall_time(saved_until)
         return True
 
-    def __pre_receive_f_init(self, apply_overlay: bool) -> None:
+    def __pre_receive_f_init(self, apply_overlay: Optional[bool]) -> None:
         """Receives on all ports connected to F_INIT.
 
         This receives all incoming messages on F_INIT and stores them
         in self._f_init_cache.
         """
+        if apply_overlay is not None:
```

Comment:
Oh hey, look at that. Never mind, this works :). We'll remove it at some point in the future.